readme.md

```diff
@@ -113,6 +113,8 @@ Examples:
   lighthouse <url> --quiet --chrome-flags="--headless"                      Launch Headless Chrome, turn off logging
   lighthouse <url> --extra-headers "{\"Cookie\":\"monster=blue\"}"          Stringify\'d JSON HTTP Header key/value pairs to send in requests
   lighthouse <url> --extra-headers=./path/to/file.json                      Path to JSON file of HTTP Header key/value pairs to send in requests
+  lighthouse <url> --only-categories=performance,pwa                        Only run specific categories. Valid categories: accessibility,
+                                                                            best-practices, performance, pwa.
```

Comment:
```suggestion
                                                                            best-practices, seo, performance, pwa.
```

Reply:
I didn't realize "seo" was an option! Thank you.

Is there a particular reason for your suggested rder, or is it ok if I maintain alphabetical order?

Reply:
Added "seo" in alphabetical order.

Reply:
alphabetical is fine. the only other acceptable order would be the order in which the categories are rendered.